doc/docs/Installation.md

```diff
@@ -270,11 +270,146 @@ By default, Meep's configure script tries to guess the gcc `-march` flag for the
 Python Bindings
 ---------------
 
-A beta version of the Python bindings for Meep (PyMeep) is available in serial (i.e., non-MPI) and parallel versions. There are two ways to obtain PyMeep.
+A beta version of the Python bindings for Meep (PyMeep) is available in serial (i.e., non-MPI) and parallel versions. There are two ways to obtain PyMeep &mdash; building from source or installing binary Conda packages.
 
 ### Building From Source
 
-You can find instructions for building from source on Ubuntu for serial PyMeep [here](https://www.mail-archive.com/meep-discuss@ab-initio.mit.edu/msg05850.html), and parallel PyMeep [here](https://www.mail-archive.com/meep-discuss@ab-initio.mit.edu/msg05884.html).
+Here are instructions for building from source on Ubuntu 16.04 for serial and parallel PyMeep.
+
+#### Serial
+
+```bash
+#!/bin/bash
+
+export LD_LIBRARY_PATH="/usr/local/lib:/usr/lib/x86_64-linux-gnu/hdf5/serial"
+export LDFLAGS="-L/usr/local/lib -L/usr/lib/x86_64-linux-gnu/hdf5/serial"
+export CPPFLAGS="-I/usr/local/include -I/usr/include/hdf5/serial"
```

Comment:
I would no longer recommend using environment variables for these things (397c693d42f924f73d6c29bcedf12c1098b4b025).  It is better to pass flags directly to `configure`.  (That way the configure script will "remember" them if you run `make` again, for example.)

Reply:
In particular, it is especially better to use `-Wl,-rpath,...` rather than `LD_LIBRARY_FLAGS`, since with `-rpath` the correct path will be embedded directly into the `meep` executable.